x-pack/plugin/data-frame/qa/single-node-tests/src/test/java/org/elasticsearch/xpack/dataframe/integration/DataFrameGetAndGetStatsIT.java

```diff
@@ -131,7 +130,6 @@ public void testGetPersistedStatsWithoutTask() throws Exception {
         for (Map<String, Object> transformStats : transformsStats) {
             Map<String, Object> stat = (Map<String, Object>)transformStats.get("stats");
             assertThat(((Integer)stat.get("documents_processed")), greaterThan(0));
-            assertThat(((Integer)stat.get("search_time_in_ms")), greaterThan(0));
```

Comment:
or use `greaterThanOrEqualTo`

Reply:
@davidkyle I could, but that does not add anything :(. A `0` value is also the default value for when there aren't stats recorded

Reply:
it proves the field is present... but that should be covered in other tests